src/modules/0.0.22/aws_codedeploy/entity/revision.ts

```diff
@@ -0,0 +1,52 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { CodedeployApplication } from './application';
+
+export enum RevisionType {
+  S3 = 'S3',
+  GITHUB = 'Github',
+}
+
+@Entity()
+export class CodedeployRevision {
+  @PrimaryGeneratedColumn()
+  @cloudId
+  id: number;
```

Comment:
yes, this I'm not so sure about, If I'm not wrong, basically for every loop in the `apply` call, for every record in the db it will think it needs to be created, because it did not encounter its counterpart, and same for every cloud record, it will think it needs to be deleted.